File: DSA/Riya_590027051/Day_3/Question2.c
#include <stdio.h>

int main() {
    int n, target, count = 0;

    // Input size of array
    scanf("%d", &n);

    int arr[n];

    // Input array elements
    for(int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    // Input target value
    scanf("%d", &target);

    // Count occurrences
    for(int i = 0; i < n; i++) {
        if(arr[i] == target) {
            count++;
        }
    }

    printf("%d\n", count);

    return 0;
}

File: DSA/DivinaNarula_590025969/Day_3/Question2.py
def count_occurrences(arr, target):
    count = 0

    for num in arr:
        if num == target:
            count += 1

    return count

input_values = input("Enter array elements separated by spaces: ").split()

arr = []
for value in input_values:
    arr.append(int(value))

target = int(input("Enter target value: "))

result = count_occurrences(arr, target)

print("Number of occurrences =", result)